api/src/main/java/com/codeforcommunity/dto/leaderboard/GetLeaderboardRequest.java

```diff
@@ -0,0 +1,36 @@
+package com.codeforcommunity.dto.leaderboard;
+
+import com.codeforcommunity.dto.ApiDto;
+import com.codeforcommunity.exceptions.HandledException;
+import java.util.ArrayList;
+import java.util.List;
+
+public class GetLeaderboardRequest extends ApiDto {
+  private Integer timePeriod;
```

Comment:
I really think we should change this field in the documentation to something that indicates that the unit is days. Even just saying timePeriodInDays isn't a bad idea because it makes it a lot harder for someone to misinterpret it

Reply:
Renamed to _previousDays_